Sources/Common/Background Queue/Type/QueueTaskMetadata.swift

```diff
@@ -19,7 +19,7 @@ public struct QueueTaskMetadata: Codable, Equatable, AutoLenses {
 internal extension QueueTaskMetadata {
     static var random: QueueTaskMetadata {
         QueueTaskMetadata(taskPersistedId: String.random,
-                          taskType: QueueTaskType.trackEvent.rawValue,
+                          taskType: String.random,
```

Comment:
same looks to me to be changing the event name to a random string in a non test file